curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element.english.md

```diff
@@ -23,7 +25,7 @@ Create a <code>p</code> element below your <code>h2</code> element, and give it
 ```yml
 tests:
   - text: Create a <code>p</code> element.
-    testString: assert(($("p").length > 0), 'Create a <code>p</code> element.');
+    testString: assert(($("p").length > 0), 'Create a valid <code>p</code> element.');
```

Comment:
Is it possible to create an invalid one?

Reply:
Like this-
`<p Hello, This is Bot></p>`